src/librustc_mir/interpret/operator.rs

```diff
@@ -347,7 +347,7 @@ impl<'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>> InterpretCx<'a, 'mir, 'tcx, M>
             }
             _ => {
                 assert!(layout.ty.is_integral());
-                let val = val.to_bits(layout.size)?;
+                let val = self.memory.force_bits(val, layout.size)?;
```

Comment:
just use `self.force_bits`, it already forwards to `self.memory.force_bits` (same below)

Reply:
going to fix this